coderedcms/blocks/__init__.py

```diff
@@ -16,7 +16,7 @@
 # Collections of blocks commonly used together.
 
 HTML_STREAMBLOCKS = [
-    ('text', blocks.RichTextBlock(icon='fa-file-text-o')),
+    ('text', blocks.RichTextBlock(icon='fa-file-text-o', template='coderedcms/blocks/rich_text_block.html')),
```

Comment:
Can you confirm this won't trigger a migration? if so include the migration.

Reply:
Actually, on second thought, what do you think about making our own richtext block (that just inherits from blocks.RichTextBlock, and specifies the template), so that anywhere it gets used will automatically have the template (and icon)?